app/views/manager/application/_navigation.html.erb

```diff
@@ -28,4 +28,7 @@ as defined by the routes in the `admin/` namespace
   <% if Rails.application.secrets.sendinblue[:enabled] && ENV["SAML_IDP_ENABLED"] == "enabled" %>
```

Comment:
pour des raisons d'homogénéité je me demande si on devrait pas faire comme pour dolist (au passage on peut virer ce enabled qui est confusant car utiliser que pr le saml sib)

Reply:
Si ça te va, je verrai bien dans une prochaine PR de virer tout ce qui concerne le SAML pour SIB (voir commentaire ci-dessous pour la raison)